### ISSUE TITLE
Start-Up crash – unable to launch fpdb

### ISSUE THREAD

**Issue by ValueVich** [philroberts/FPDB-for-OSX#38]
It just bounces real quick in the dock and disappears, unable to launch.

I'm using OS X 10.11.4

I had an older version of fpdb wich worked fine. but now it doesn't even launch anymore... what should I do?

**Comment by philroberts** [comment]
To clarify, you still have the old version, but it doesn't work either with the same symptoms?

If you can find the log files for me (they should be in ~/.fpdb/logs) there might be some relevant info there.

It sounds like the new version might have done something to your database/config file, so you could also try moving the ~/.fpdb folder to a backup location and running FPDB again.  It should create a fresh folder with a clean config and database.

**Comment by ValueVich** [comment]
Hi, thank for the info, I got it to work now with the workaround of deleting the hole .fpdb directory and just giving it a fresh install.

However, I have found a missing feature/bug:
when you are using the handreplayer for rewatching the hand, I found that the spitcard is missing in the replayer (for replaying Courchevel games e.g.) – should this be posted as a new issue, or is it ok if I leave it her? Hope this helps, thx for the fine program.
